tutorials-and-examples/hf-tgi/main.tf

```diff
@@ -87,6 +87,16 @@ resource "kubernetes_deployment" "inference_deployment" {
             value = "mistralai/Mistral-7B-Instruct-v0.1"
           }
 
+          env {
```

Comment:
should we add comments explaining why these values are necessary?

Reply:
done